src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -195,6 +196,10 @@
  *     <td style="text-align:center;">{@link MemorySegment}</td>
  * </tbody>
  * </table></blockquote>
+ * <p>
+ * Note that due to limited ABI specification coverage, none of the native linker implementations supports
```

Comment:
Is there something bigger to say here? E.g. can I pass a C int with an alignment other than its size? E.g. is there a requirement that the alignment of _all_ layouts should be identical to their natural alignment? If so, we should say so.

Reply:
I've added a list of constraints to the linker class javadoc which covers this.